src/bundle/Resources/richtext/schemas/docbook/ezpublish.rng

```diff
@@ -75,6 +75,19 @@
         </oneOrMore>
       </element>
     </define>
+    <define name="db.itemizedlist.mark.attribute">
+      <attribute name="mark">
+        <a:documentation>Identifies the type of mark to be used on items in this list</a:documentation>
+        <choice>
+          <value>disc</value>
+          <a:documentation>Items in  the list should be marked using a disc</a:documentation>
+          <value>circle</value>
+          <a:documentation>Items in  the list should be marked using a circle</a:documentation>
+          <value>square</value>
+          <a:documentation>Items in  the list should be marked using a square</a:documentation>
+        </choice>
+      </attribute>
+    </define>
```

Comment:
No need to define it, it's defined in `docbook.rng` which is loaded by ezpublish.rng

Reply:
`mark` is in docbook defined as `NMTOKEN` ( string but no whitespaces). It is [suggested in DocBook](https://tdg.docbook.org/tdg/5.0/itemizedlist.html) ref doc to limit the number of accepted values. So that is what I am doing here. If you really don't want it, I can remove it
> In order to enforce a standard set of marks at your organization, it may be useful to construct a customization layer that limits the values of the mark attribute to an enumerated list.

Reply:
Removed in f7d30bfbc903411207f2739be293b5af84fd4a8a